lib/create-license.js

```diff
@@ -2,10 +2,9 @@ import getTemplateFileContent from "./get-template-file-content.js";
 import writePrettyFile from "./write-pretty-file.js";
 
 export default async function createLicense(name) {
-  await writePrettyFile(
-    "LICENSE.md",
-    getTemplateFileContent("LICENSE.md")
-      .replace("{{YEAR}}", new Date().getFullYear())
-      .replace("{{OWNER}}", name),
-  );
+  const templateContent = await getTemplateFileContent("LICENSE.md");
+  const content = templateContent
+    .replace("{{YEAR}}", new Date().getFullYear())
+    .replace("{{OWNER}}", name);
+  await writePrettyFile("LICENSE.md", content);
```

Comment:
```suggestion
  await writePrettyFile(
    "LICENSE.md",
    getTemplateFileContent("LICENSE.md")
      .replace("{{YEAR}}", new Date().getFullYear())
      .replace("{{OWNER}}", name),
  );
```
Please revert these changes, `getTemplteFileContent()` is not async